libsolidity/interface/Natspec.h

```diff
@@ -60,6 +61,7 @@ class Natspec
 	/// @return      A JSON representation
 	///              of the contract's developer documentation
 	static Json::Value devDocumentation(std::multimap<std::string, DocTag> const& _tags);
+	static Json::Value extractReturnParameterDocs(std::multimap<std::string, DocTag> const& _tags, FunctionDefinition const& _functionDef);
```

Comment:
This could use some more documentation, perhaps similar to the function ``devDocumentation`` above.